pyproject.toml

```diff
@@ -1,3 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "django-stubs"
+version = "5.2.0"
+description = "Mypy stubs for Django"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+requires-python = ">=3.10"
+authors = [
+    { name = "Maksim Kurnikov", email = "maxim.kurnikov@gmail.com" },
+]
+maintainers = [
+    { name = "Marti Raudsepp", email = "marti@juffo.org" },
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

Comment:
I believe that the license trove classifiers are deprecated in favour of an SPDX license expression in `license`, so this could be removed:

```suggestion
```